src/openrct2/park/ParkFile.h

```diff
@@ -11,10 +11,10 @@ namespace OpenRCT2
     struct GameState_t;
 
     // Current version that is saved.
-    constexpr uint32_t PARK_FILE_CURRENT_VERSION = 33;
+    constexpr uint32_t PARK_FILE_CURRENT_VERSION = 34;
```

Comment:
Why would this be necessary?

Reply:
The new flags don't exist in OpenRCT2 versions that can open park v33 files, which could break some parks once they're being used.

Reply:
How will it break? Older versions will simply ignore the flag and continue to animate them, right?

If the general opinion is that this warrants a bump, then only `PARK_FILE_CURRENT_VERSION` should be increased, as older versions _can_ still open it.

Reply:
While I can understand wanting to be hesitant regarding incrementing the park version, I think we should do it. Otherwise I can expect bug reports appearing from communities like DKMP whom have contests every month with potential frozen peep scenes. Not everyone there updates to the latest develop every day, which will cause confusing/frustration when those frozen peep scenes made in newer versions suddenly start walking on older versions, 😅

Reply:
Ok then, but the reason I mentioned, I think that _only_ PARK_FILE_CURRENT_VERSION should be raised, with the other left as-is.

Reply:
After some discussion on Discord, I have reverted the change to `PARK_FILE_MIN_VERSION`.